src/main/kotlin/exercise3/task1/BalancedBrackets.kt

```diff
@@ -26,7 +26,25 @@ package exercise3.task1
 
 
 internal fun isExpressionBalanced(expression: String): Boolean {
-    TODO("Implement me!!!")
+    val bracketsMap = mapOf(
+        '(' to ')',
+        '[' to ']',
+        '{' to '}'
+    )
+    val openingBrackets = bracketsMap.keys
+    val closingBrackets = bracketsMap.values
+
+    return expression.fold(emptyList<Char>()) { stack, char ->
```

Comment:
Great job using the `fold` function! 👏🏻  This is also the type of solution I would apply.